app/actions/queries.js

```diff
@@ -31,7 +31,7 @@ export function getAvgMessagesByHour({ year, month, day }) {
           r.row('ts').date().eq(r.time(year, month, day, 'Z'))
         )
         .group(r.row('ts').hours())
-        .avg('score')
+        .count()
```

Comment:
Replacing `avg` with `count` means this function no longer does what it's name implies and will break any features that want the average. Could we make a separate query that gets the count instead?